lms/djangoapps/teams/static/teams/js/spec_helpers/team_discussion_helpers.js

```diff
@@ -0,0 +1,126 @@
+define([
+    'underscore', 'common/js/spec_helpers/ajax_helpers'
+], function (_, AjaxHelpers) {
+    'use strict';
+    var createMockPostResponse, createMockDiscussionResponse, createAnnotatedContentInfo, createMockThreadResponse,
+        testCourseID = 'course/1',
+        testUser = 'testUser',
+        testTeamDiscussionID = "12345";
+
+    createMockPostResponse = function(options) {
+        return _.extend(_.extend({
```

Comment:
Regarding the mock data: I'll try to trim it down and see what the minimum is that is required. This is the full set of data returned by a real response which seems unduly verbose.